go-controller/pkg/ovn/ovn.go

```diff
@@ -798,8 +800,9 @@ func (oc *Controller) WatchNamespaces() {
 	// TODO(FF): make this function return failures
 	if err != nil {
 		klog.Errorf("Failed to watch namespaces err: %v", err)
+		// return err
```

Comment:
dont you need to return the error here?

Reply:
yes, absolutely!
```
// This function implements the main body of work of syncNamespaces.
// Upon failure, it may be invoked multiple times in order to avoid a pod restart.
```
I was a little too conservative on that one. 🤦